src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/TypeDescriptor.cs

```diff
@@ -2919,10 +2926,11 @@ private sealed class ComNativeDescriptorProxy : TypeDescriptionProvider
         {
             private readonly TypeDescriptionProvider _comNativeDescriptor;
 
+            [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2072:UnrecognizedReflectionPattern",
+                Justification = "The trimmer can't find the ComNativeDescriptor type because System.Windows.Forms isn't available.")]
             public ComNativeDescriptorProxy()
             {
-                Assembly assembly = Assembly.Load("System.Windows.Forms");
-                Type realComNativeDescriptor = assembly.GetType("System.Windows.Forms.ComponentModel.Com2Interop.ComNativeDescriptor", throwOnError: true);
+                Type realComNativeDescriptor = Type.GetType("System.Windows.Forms.ComponentModel.Com2Interop.ComNativeDescriptor, System.Windows.Forms", throwOnError: true);
```

Comment:
FYI - @AaronRobinsonMSFT. I changed this pattern given the discussion at https://github.com/mono/linker/issues/1884#issuecomment-796734517.

Let me know if you have feedback on this change. And how I can test it to make sure I didn't break the scenario.

Reply:
This looks fine to me. Adding @JeremyKuhne, @RussKie, and @merriemcgaw for WinForms impact.

Reply:
Should be ok? I've never fully walked through this though. :(